packages/libs/src/sdk/services/Auth/constants.ts

```diff
@@ -0,0 +1,24 @@
+import type { LocalStorage } from '@audius/hedgehog'
+
+import { productionConfig } from '../../config'
+
+import { AuthConfig } from './types'
+
+/**
+ * Fallback for localStorage that works in node and the browser
+ * @returns localStorage
+ */
+export const getPlatformLocalStorage = async (): Promise<LocalStorage> => {
```

Comment:
constants.ts seems like a weird place for `getPlatformLocalStorage`

Reply:
made a util 👍